.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--
+  Thanks for submitting a pull request!
+  We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request.
+
+  Please mark this PR as "Draft" if it is not ready for review.
+-->
+
+### Summary
+
+<!--
+ Explain the **motivation** for making this change. Why was this change necessary?
+ What existing problem does the pull request solve?
+ Any implementation details to explain? What code paths or UI flow do the code changes hit?
+-->
+
+### Steps to test
+
+<!--
+  How exactly did you verify that your PR solves the issue?
+  If applicable, share the steps that a reviewer should follow to validate the new behavior.
+  Example: The exact commands you ran and their output, screenshots / gifs / videos if the pull request changes the UI.
+-->
```

Comment:
Recently seeing some tags on PR's. Do we need some rules around those?

<img width="937" alt="image" src="https://user-images.githubusercontent.com/7558499/204733767-d9436078-3835-4648-8618-9784313de981.png">

Reply:
we can add automation for type mapping from PR title
but i think we can use it by our own judgment until more structure is needed

Reply:
Agree with Fionna, I think common sense is enough here